src/evm/protocol/filters.rs

```diff
@@ -169,6 +169,26 @@ pub fn uniswap_v4_pool_with_hook_filter(component: &ComponentWithState) -> bool
     true
 }
 
+pub fn angstrom_pool_filter(component: &ComponentWithState) -> bool {
+    const ANGSTROM_ADDRESS: alloy::primitives::Address =
+        alloy::primitives::address!("0x0000000aa232009084Bd71A5797d089AA4Edfad4");
+
+    if let Some(hooks) = component
+        .component
+        .static_attributes
+        .get("hooks")
+    {
+        if hooks.to_vec() != **ANGSTROM_ADDRESS {
+            debug!(
+                "Filtering out UniswapV4 pool {} because it is not angstrom",
+                component.component.id
+            );
+            return false;
+        }
+    }
+    true
+}
```

Comment:
Most likely we won't need this. Maybe we can have the ANGSTROM_ADDRESS constant somewhere exposed.